packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -971,7 +979,7 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
             fileUri = new URI(node.fileUri);
         }
 
-        const opts: EditorOpenerOptions | undefined = !DiffUris.isDiffUri(fileUri) ? {
+        let opts: EditorOpenerOptions | undefined = !DiffUris.isDiffUri(fileUri) ? {
```

Comment:
The conditions for deciding whether to show a diff widget (l. 974 - GitHub won't let me comment on old code) produce some weird behavior. Single clicking will open a diff widget, but double-clicking will open a normal editor. That isn't consistent with VSCode, but might be desirable. VSCode has read-write diff widgets where you can edit the right side, but we don't (seem to), so if you want to edit the file, you need a way to open it in a normal editor. Still, if the focus of the PR is fixing the preview vs. non-preview behavior, it might be worth seeing whether we can closer to VSCode's behavior.